src/ucp/tag/rndv.c

```diff
@@ -305,8 +319,12 @@ UCS_PROFILE_FUNC(ucs_status_t, ucp_proto_progress_rndv_get_zcopy, (self),
         if (rndv_req->send.state.uct_comp.count == 0) {
             ucp_rndv_complete_rndv_get(rndv_req);
         }
-        return UCS_OK;
+    } else if (status == UCS_OK) {
```

Comment:
is this a fix for case of error from uct_ep_get_zcopy()?

Reply:
this is fix for return OK status too early: in case if we have to split packet by 2 chunks (for alignment for instance), and first zcopy is completed immediately with OK status - do not allow progress to complete this request